File: src/tests/stuff/moretest/dict_perf.go
package main

import "fmt"

func main() {
    fmt.Println("dict_perf")
    d := make(map[int]int)
    s := 0
    for i := 0; i < 10000; i++ {
        d[i] = i * 2
    }
    for i := 0; i < 10000; i++ {
        s += d[i]
    }
    fmt.Println(s)
}

File: src/tests/stuff/moretest/dict_perf.rs
use std::collections::HashMap;
fn main() {
    println!("dict_perf");
    let mut d = HashMap::new();
    let mut s: i64 = 0;
    for i in 0..10000 { d.insert(i, i * 2); }
    for i in 0..10000 { s += d[&i]; }
    println!("{}", s);
}
